src/lwc/geFormRenderer/geFormRenderer.js

```diff
@@ -761,19 +833,27 @@ export default class GeFormRenderer extends NavigationMixin(LightningElement) {
         this.widgetData = {...this.widgetData, ...payload};
     }
 
-    getData(sections) {
-        let { diRecord, widgetValues } =
-            GeFormService.getDataImportRecord(sections);
-
-        if (!diRecord[NPSP_DATA_IMPORT_BATCH_FIELD.fieldApiName]) {
-            diRecord[NPSP_DATA_IMPORT_BATCH_FIELD.fieldApiName] = this.batchId;
+    /*******************************************************************************
+    * @description Builds a full DataImport__c record from the provided form sections
+    * and potential donor data selected from the review donations modal.
+    *
+    * @param {list} sections: List of all form sections
+    * @param {object} dataImportWithDonorData: Object holding data import values from
+    * the 'Review Donations' modal.
+    */
+    buildDataImportFromSections(sections, dataImportWithDonorData) {
+        let dataImportRecord =
+            GeFormService.buildDataImportRecord(sections, dataImportWithDonorData);
```

Comment:
Can buildDataImportRecord be imported here directly rather than imported by and referenced in the GeFormService cmp?

Reply:
Yup! Moved directly into `geFormRenderer` since it was referenced in this one place anyway.